### ISSUE TITLE
<Nation(s) Affected> - <Summary of In-game Crash>

### ISSUE THREAD

**Issue by trif014** [Darkest-Hour-Hoi4/Darkest-Hour-Issues#135]
### OS

Windows 11

### Any Expansions?

Together for Victory, Death or Dishonor, Waking the Tiger, Man the Guns, La Résistance, Battle For The Bosporus, No Step Back

### Are you using Steam?

Yes i am

### Were you playing in multiplayer?

No i wasn't

### Hoi4 Version

1.12.14

### DH Version (Current one is 1.0.4-hotfix1)

1.0.4-hotfix1

### Other Mods

Blue Marble - Darkest Hour Map Style | Songs of Our Darkest Hour | State Transfer Tool | Toolpack without the errors

### Description

```bash
Game crashes when I zoom in on Yugoslavia on the map paused or unpaused. Doesn't happen anywhere else.
```

### Reproduction steps

```bash
Zoom in on Yugoslavia
```

### Logs

[hoi4_20230620_210144.zip](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11812111/hoi4_20230620_210144.zip)

### Savegame

[autosave.zip](https://github.com/Darkest-Hour-Hoi4/Darkest-Hour-Issues/files/11812112/autosave.zip)

**Comment by Darijus11** [comment]
Why are you using Blue Marble?

**Comment by Darijus11** [comment]
We've already identified the issue and fixed it. We will deploy the fix with 1.0.5 version. The 1.0.5 version will be deployed when ready. In the meantime, keep 3D units off.
